lua/ulx/modules/sh/freeze.lua

```diff
@@ -0,0 +1,50 @@
+CATEGORY_NAME = "CFC"
+
+do
+    local function freezeProps( callingPlayer, targetPlayers ) 
+        local entits = ents.FindByClass( "prop_*" )
+        local entCount = 0
+        
+        for _, ent in pairs( entits ) do
+            if table.HasValue( targetPlayers, ent:CPPIGetOwner() ) then 
+                local physicsObj = ent:GetPhysicsObject()
+                if IsValid( physicsObj ) then
+                    physicsObj:EnableMotion( false )
+                    physicsObj:Sleep()
+                    entCount = entCount + 1
+                end
+            end
+        end
+
+        ulx.fancyLogAdmin( callingPlayer, "#A froze all "..entCount.." props owned by #T", targetPlayers )
+    end
+
+    local entits = ulx.command( CATEGORY_NAME, "ulx freezeprops", freezeProps, "!freezeprops" )
+    entits:addParam{ type=ULib.cmds.PlayersArg }
+    entits:defaultAccess( ULib.ACCESS_ADMIN )
+    entits:help( "Freezes target(s) props" )
+end
+
+do
+    local function freezeSelfProps( callingPlayer ) 
+        local entits = ents.FindByClass( "prop_*" )
+        local entCount = 0
+        
+        for _, ent in pairs( entits ) do
```

Comment:
if we decide to keep 2 commands instead of repeating the code in the 2 functions it could be put into its own function
```freezePlayerProps( ply )``` maybe?

Reply:
I think ill remove the second one.